Packages/Netherlands3D/WebServiceReader/Runtime/Scripts/Layer/WMSImageLayer.cs

```diff
@@ -87,26 +119,44 @@ private Tile CreateNewTile(Vector2Int tileKey)
             tile.gameObject.layer = tile.gameObject.transform.parent.gameObject.layer;
             Vector2Int origin = new Vector2Int(tileKey.x+(tileSize/2), tileKey.y + (tileSize / 2));
             tile.gameObject.transform.position = CoordConvert.RDtoUnity(origin);
-            
+            var sizeVector = new Vector3(tileSize, tileSize, tileSize);
+
+            if (prefabIsDecalProjector)
```

Comment:
Het lijkt alsof dit niet hier hoort; ik zie wel meer checks of de prefab een decal projector is en ik krijg het idee dat een service of gateway mist; of dat de 'normale' projector eigenlijk ook een losse adapter zou moeten zijn zodat ze beide dezelfde methodes hebben / interface?